README.md

```diff
@@ -3,66 +3,171 @@
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/e61f4ea81de14b289fbf83d2b0da0c35)](https://app.codacy.com/app/FrancescoBorzi/azerothcore-wotlk?utm_source=github.com&utm_medium=referral&utm_content=azerothcore/azerothcore-wotlk&utm_campaign=Badge_Grade_Settings)
 [![Master Build Status](https://travis-ci.org/azerothcore/azerothcore-wotlk.svg)](https://travis-ci.org/azerothcore/azerothcore-wotlk)
 <a href="https://scan.coverity.com/projects/azerothcore-azerothcore-wotlk">
-  <img alt="Coverity Scan Build Status"
-       src="https://scan.coverity.com/projects/13814/badge.svg"/>
+  <img alt="Coverity Scan Build Status"  src="https://scan.coverity.com/projects/13814/badge.svg"/>
 </a>
 [![Bountysource](https://www.bountysource.com/badge/tracker?tracker_id=40032087)](https://www.bountysource.com/teams/azerothcore/bounties)
 [![StackOverflow](http://img.shields.io/badge/stackoverflow-azerothcore-blue.svg)]( https://stackoverflow.com/questions/tagged/azerothcore?sort=newest)
-[![Discord](https://img.shields.io/discord/217589275766685707.svg)](https://discord.gg/gXNKCG5)
+[![Discord](https://img.shields.io/discord/217589275766685707.svg)][discord]
 
 
-## Introduction
+## INTRODUCTION
 
-AzerothCore is an open source game-server application for World of Warcraft, currently supporting the 3.3.5a game version.
+AzerothCore (AC) is an open source game-server application for World of Warcraft, currently supporting the 3.3.5a game version.
 
-It is written in C++ and based on MaNGOS, TrinityCore and SunwellCore.
+It is written in C++ and is based on MaNGOS, TrinityCore and SunwellCore.
 
-:moneybag: You can support the project by financing the resolution of issues [using Bountysource](http://www.azerothcore.org/wiki/Bountysource).
 
-## Why AzerothCore?
+## WHY AZEROTHCORE?
 
-Our main goal is to create a playable game server, offering a fully working game experience. In short, we focus on the **user experience (UX)**.
+1. Stability
+1. Authenticity of the content
+1. [Modularity](https://en.wikipedia.org/wiki/Modular_programming)
+1. A lot of modules to choose from
+1. Better configuration files system
+1. Compatibility with other emulators
+1. Friendly and helpful community
 
-We also believe in [software modularity](https://en.wikipedia.org/wiki/Modular_programming), because the more the software is modular, and the more it will be easy to be maintained, understood, developed, etc... So we organized the structure of AzerothCore as modular as possible, giving the possibility to [create new modules](http://www.azerothcore.org/wiki/Create-a-Module) to add custom features (instead of patching the core).
+#### Stability
 
-## Learning project
+As players and administrators ourselves, we take great care into the stability of our core. Other projects focus on the development side of things and tend to forget that users want stability above everything else.
```

Comment:
we might add something about testing here, e.g. "No one is pushing commits directly to the core. All changes are **reviewed** and **tested** before they get to the `master` branch."

Reply:
ok doing it